store/sqlstore/post_store.go

```diff
@@ -1312,42 +1312,73 @@ func (s *SqlPostStore) GetMaxPostSize() int {
 }
 
 func (s *SqlPostStore) GetParentsForExportAfter(limit int, afterId string) ([]*model.PostForExport, *model.AppError) {
-	var posts []*model.PostForExport
-	_, err := s.GetSearchReplica().Select(&posts, `
-                SELECT
-                    p1.*,
-                    Users.Username as Username,
-                    Teams.Name as TeamName,
-                    Channels.Name as ChannelName
-                FROM
-                    Posts p1
-                INNER JOIN
-                    Channels ON p1.ChannelId = Channels.Id
-                INNER JOIN
-                    Teams ON Channels.TeamId = Teams.Id
-                INNER JOIN
-                    Users ON p1.UserId = Users.Id
-                WHERE
-                    p1.Id > :AfterId
-                    AND p1.ParentId = ''
-                    AND p1.DeleteAt = 0
-					AND Channels.DeleteAt = 0
-					AND Teams.DeleteAt = 0
-                ORDER BY
-                    p1.Id
-                LIMIT
-                    :Limit`,
-		map[string]interface{}{"Limit": limit, "AfterId": afterId})
+	for {
+		var parentPosts []*model.Post
+		_, err := s.GetReplica().Select(&parentPosts,
+			`SELECT
+				*
+			FROM
+				Posts
+			WHERE
+				Id > :AfterId
+				AND RootId = ''
```

Comment:
I'm confused about which one is deprecated: `ParentId` or `RootId`?

Reply:
From what I have seen around the code and was told, `RootId` is indexed and is what should be used.

Reply:
OK great. I'll remember that :)